lomap/graphgen.py

```diff
@@ -106,7 +111,8 @@ def __init__(self, dbase):
         self.max_nodes = 100
 
         # The maximum threshold distance in A unit used to select if a molecule is depicted
```

Comment:
Not your fault, but can we make this "angstroms" rather than "A"? It took me a minute to figure out what this was saying. (Actually, I still don't totally understand it -- is it saying that molecules that are more than `max_mol_size` angstroms in some dimension won't be depicted?)